main/src/ca/uwaterloo/flix/language/phase/Parser2.scala

```diff
@@ -1883,10 +1883,7 @@ object Parser2 {
       val mark = open()
       expect(TokenKind.KeywordXvar)
       nameUnqualified(NAME_TAG)
-      expect(TokenKind.ParenL)
-      // TODO: Ext-Variants: Limited to one expression.
-      expression()
-      expect(TokenKind.ParenR)
+      exprDelimited()
```

Comment:
Hmm this seems a bit tooo lenient to me. Can you make it parse the argument list directly? similar to Exp.Apply
```
        nth(0) match {
          case TokenKind.ParenL => // Function call.
            val mark = openBefore(lhs)
            arguments()
            lhs = close(mark, TreeKind.Expr.Apply)
            lhs = close(openBefore(lhs), TreeKind.Expr.Expr)
```

Reply:
if you have `xvar Example q[i]` the error will be that `q[i]` is not a tuple. Whereas the more direct parsing would say `expected '(', found q` or something like that

current solution might also allow `xvar Example (2)`